src/it/basic/build.xml

```diff
@@ -33,10 +35,20 @@
 
         <wlp:install-feature ref="testServer" name="mongodb-2.0" whenFileExists="ignore" acceptLicense="true" />
         <!-- install the same feature twice - should not cause an error -->
-        <wlp:install-feature ref="testServer" name="mongodb-2.0" whenFileExists="ignore" acceptLicense="true" />
-
+        <wlp:install-feature ref="testServer" whenFileExists="ignore" acceptLicense="true">
```

Comment:
But we've lost a test case for the name="mongodb-2.0" case, as the test casn now does the nested element way instead.